cfme/common/vm.py

```diff
@@ -316,8 +316,7 @@ def find_quadicon(
             else:
                 raise TemplateNotFound("Template '{}' not found in UI!".format(self.name))
 
-        # this is causing some issues in 5.5.0.9, commenting out for a bit
-        # paginator.results_per_page(1000)
+        paginator.results_per_page(1000)
```

Comment:
Well... ideally we would pull this from a collection and collections need to know how to paginate, correct me if I'm wrong.